### PR TITLE
basic unit test

### PR DESCRIPTION
**Pull request by peterbe**
It's really quick and just getting us started. I don't want to perfect the test yet. I'd rather land something, wire up the CircleCI + Dockerhub and then we can perfect the tests later. And if we forget, it probably wasn't important.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,11 @@ WORKDIR /app
 
 COPY ./setup.py /app/setup.py
 COPY ./README.md /app/README.md
-RUN pip install -e ".[dev]"
+COPY ./run.sh /app/run.sh
+COPY ./pytest.ini /app/pytest.ini
+RUN pip install -e ".[dev,test]"
 COPY ./main.py /app/main.py
+COPY ./test_main.py /app/test_main.py
```
Comment (peterbe) [inline Dockerfile]: With all these `COPY`s, it's it better to put all the files in a `src/` directory or something?

Comment (leplatrem) [reply Dockerfile]: `COPY . /app` with appropriate `.dockerignore` ?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,26 @@ Another useful variable is `DEBUG=true`. If you use this, all repeated network
 requests are cached to disk. Note! If you want to invalidate that cache, delete
 the file `requests_cache1.sqlite`.
 
+## Running tests
+
+The simplest form is to run:
+
+```bash
+$ pip install -e ".[devtest]"
```
Comment (leplatrem) [inline README.md]: `[dev,test]`

File: Dockerfile (hunk 1 above)
Comment (peterbe) [reply Dockerfile]: Lemme try.

**Comment by peterbe** [comment]
@leplatrem One last check. I went with the `COPY . /app` and I copied the `.dockerignore` from the `.gitignore`.

**Pull request merged**